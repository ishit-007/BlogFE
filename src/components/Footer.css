footer {
  flex:none;
  display: flex;
  color: white;
  background-color: black;
  width: 100%;
}
.copyright {
  margin-left: 7rem;
  padding-top: 1.5rem;
  padding-bottom: 1.5rem;
}
.footerText {
  padding-top: 1.5rem;
  padding-bottom: 1.5rem;
}
