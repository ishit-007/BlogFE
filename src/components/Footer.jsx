import React from 'react';
import './Footer.css';
const Footer = ()=> {
  return (
    <div>
      <footer>
        <div className="copyright">&copy;</div>
        <div className="footerText">artifact.com 2019</div>
      </footer>
    </div>
  );
};
export default Footer;
