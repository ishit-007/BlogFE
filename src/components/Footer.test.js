import Footer from './Footer'
import React from 'react'
import { render } from '@testing-library/react'
import renderer from 'react-test-renderer'

describe('Footer', () => {
    it('should render the Footer component', async() => {
        const tree=renderer.create(<Footer/>).toJSON();
        expect(tree).toMatchSnapshot();
    });
});