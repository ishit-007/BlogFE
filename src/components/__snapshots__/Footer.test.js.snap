// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render the Footer component 1`] = `
<div>
  <footer>
    <div
      className="copyright"
    >
      Â©
    </div>
    <div
      className="footerText"
    >
      artifact.com 2019
    </div>
  </footer>
</div>
`;
